ResearchKitTests/ORKAccessibilityTests.m

```diff
@@ -39,11 +39,9 @@ @implementation ORKAccessibilityTests
 
 - (void)setUp {
     [super setUp];
-    // Put setup code here. This method is called before the invocation of each test method in the class.
 }
```

Comment:
Could probably remove all these calls too, no need to call them if nothing else is being done.